src/NServiceBus.Transport.RabbitMQ.CommandLine/ConnectionFactoryBinder.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace NServiceBus.Transport.RabbitMQ.CommandLine
+{
+    using System.CommandLine;
+    using System.CommandLine.Binding;
+    using System.Security.Cryptography.X509Certificates;
+
+    class ConnectionFactoryBinder : BinderBase<ConnectionFactory>
+    {
+        readonly Option<string> connectionStringOption;
```

Comment:
nitpick: should we move the fields to the bottom of the class to align with the other classes?